### DISCUSSION TITLE
5.0的DLedger Controller模式和master-slave在不考虑主备自动切换的情况下，那个性能比较高呐？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [apache/rocketmq#6997] [General]
5.0的DLedger Controller模式和master-slave在不考虑主备自动切换的情况下，那个性能比较高呐？

**Comment by RongtongJin** [comment]
> 5.0的DLedger Controller模式和master-slave在不考虑主备自动切换的情况下，那个性能比较高呐？

这两个性能应该几乎是一致的。切换功能不影响存储以及复制。
